scripts/aznfs_install.sh

```diff
@@ -299,6 +299,11 @@ ensure_pkg()
     elif [ "$distro" == "sles" ]; then
         zypper=1
         zypper install -y $pkg
+    else
+        eecho "[FATAL] Unknown linux distro"
+        pecho "Download .deb/.rpm package based on your distro from 'https://github.com/Azure/AZNFS-mount/releases/latest'"
```

Comment:
this error should be
eecho "[FATAL] Unsupported linux distro <$distro>"
pecho "Check 'https://github.com/Azure/AZNFS-mount/blob/main/README.md#supported-distros' to see the list of supported distros, or"
pecho "Download .deb/.rpm package based on your distro from 'https://github.com/Azure/AZNFS-mount/releases/latest'"

Reply:
Does it mean now we will fail for distros like oracle linux, which we used to work before?